docs/eventing/migration/sinkbinding.md

```diff
@@ -1,88 +1,97 @@
 ---
-title: "Migrating from ContainerSource to SinkBinding"
+title: "SinkBinding alternative to ContainerSource"
 weight: 20
 type: "docs"
 aliases:
-   - /docs/eventing/ping.md
+   - /docs/eventing/sinkbinding.md
 ---
 
-The deprecated `ContainerSource` should be converted to a `SinkBinding`.
+`ContainerSource` can be seen as the combination of
+`SinkBinding`+`Deployment`. In fact, that
+is exactly how it's implemented!
 
-The YAML file for a `ContainerSource` that emits events to a Knative Serving service will look similar to this:
+In YAML, these two options are equivalent:
 
-```yaml
-apiVersion: sources.eventing.knative.dev/v1alpha1
-kind: ContainerSource
-metadata:
-  name: urbanobservatory-event-source
-spec:
-  image: quay.io/openshift-knative/knative-eventing-sources-websocketsource:latest
-  args:
-    - '--source=wss://api.usb.urbanobservatory.ac.uk/stream'
-    - '--eventType=my.custom.event'
-  sink:
-    apiVersion: serving.knative.dev/v1
-    kind: Service
-    name: wss-event-display
-```
+1. `ContainerSource` that emits events to a `Knative Service`:
 
-The referenced image of the `ContainerSource` needed an "sink" argument, [see](https://knative.dev/docs/eventing/#containersource) for details.
+    ```yaml
+    apiVersion: sources.knative.dev/v1alpha2
+    kind: ContainerSource
+    metadata:
+      name: urbanobservatory-event-source
+    spec:
+      template:
+        spec:
+          containers:
+          - image: quay.io/openshift-knative/knative-eventing-sources-websocketsource:latest
+            args:
+            - '--source=wss://api.usb.urbanobservatory.ac.uk/stream'
+            - '--eventType=my.custom.event'
+      sink:
+        ref:
+          apiVersion: serving.knative.dev/v1
+          kind: Service
+          name: wss-event-display
+    ```
 
-To migrate this source to a `SinkBinding`, a few steps are required. Instead of using a `ContainerSource`,
-you need to create a `SinkBinding`, like:
+2. `SinkBinding` + `Deployment`. Something like the following:
 
-```yaml
-apiVersion: sources.knative.dev/v1alpha2
-kind: SinkBinding
-metadata:
-  name: bind-wss
-spec:
-  subject:
-    apiVersion: apps/v1
-    kind: Deployment
-    selector:
-      matchLabels:
-        app: wss
-  sink:
-    ref:
-      apiVersion: serving.knative.dev/v1
-      kind: Service
-      name: wss-event-display
-```
+    ```yaml
+    apiVersion: sources.knative.dev/v1alpha2
+    kind: SinkBinding
+    metadata:
+      name: bind-wss
+    spec:
+      subject:
+        apiVersion: apps/v1
+        kind: Deployment
+        selector:
+          matchLabels:
+            app: wss
+      sink:
+        ref:
+          apiVersion: serving.knative.dev/v1
+          kind: Service
+          name: wss-event-display
+    ```
 
-Here the `SinkBinding`'s `subject` references to a Kubernetes `Deployment`, that is labeled with `app: wss`. The YAML for the `Deployment`
-looks like:
+    Here the `SinkBinding`'s `subject` references to a Kubernetes
```

Comment:
do you want to say something about the sink binding selection mode here?

Reply:
done, added some clarification! Thanks!